flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/functions/hive/conversion/HiveInspectors.java

```diff
@@ -436,6 +440,7 @@ public static DataType toFlinkType(ObjectInspector inspector) {
 					case DATE:
 						return Date.class;
 					case TIMESTAMP:
+						return LocalDateTime.class;
```

Comment:
why is this LocalDateTime.class rather than Timestamp.class?

Reply:
Flink timestamp by default bridges to LocalDateTime